components/kyma-environment-broker/internal/process/deprovisioning/avs_evaluations_test.go

```diff
@@ -71,15 +82,31 @@ func newMockAvsOauthServer() *httptest.Server {
 }
 
 func newMockAvsServer(t *testing.T) *httptest.Server {
-	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	router := mux.NewRouter()
+	router.HandleFunc("/{evalId}", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, r.Method, http.MethodDelete)
+		vars := mux.Vars(r)
+		evalIdsHolder = append(evalIdsHolder, extractId(vars, "evalId", t))
+		w.WriteHeader(http.StatusOK)
+	}))
```

Comment:
you can simply add:
```go
router.HandleFunc("/{evalId}", func(w http.ResponseWriter, r *http.Request) {
	vars := mux.Vars(r)
	evalIdsHolder = append(evalIdsHolder, extractId(vars, "evalId", t))
	w.WriteHeader(http.StatusOK)
}).Methods(http.MethodDelete)
```
then your testServer will return 404 when you call e.g. PUT `/{evalId}`

Reply:
applied the feedback